tests/client/core/test_utils.js

```diff
@@ -516,7 +512,7 @@ describe('loadCategoriesIfNeeded', () => {
     return loadCategoriesIfNeeded(props).then(() => {
       assert(dispatch.calledWith(action), 'dispatch not called');
       mockApi.verify();
-      mockActions.verify();
+      return mockActions.verify();
```

Comment:
Does this eslint rule enforce the return of the original promise *and* returns in the `then()`?